cross/domoticz/Makefile

```diff
@@ -1,36 +1,33 @@
 PKG_NAME = domoticz
-PKG_VERS = $(PKG_SVN_REV)
+PKG_VERS = 4.9700
 PKG_EXT = tar.gz
-PKG_DOWNLOAD_METHOD = svn
-PKG_SVN_REV = 2284
-PKG_DIST_NAME = trunk
-PKG_DIST_SITE = svn://svn.code.sf.net/p/domoticz/code
-PKG_DIST_FILE = $(PKG_NAME)-r$(PKG_SVN_REV).$(PKG_EXT)
-PKG_DIR = $(PKG_NAME)-r*
+PKG_DIST_NAME = $(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/domoticz/domoticz/archive
+PKG_DIST_FILE = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
 
-DEPENDS = cross/boost cross/sqlite cross/curl
+DEPENDS = cross/boost cross/sqlite cross/curl cross/python3
 
 HOMEPAGE = http://www.domoticz.com/
-COMMENT  = Domoticz is a Home Automation System
-LICENSE  = 
+COMMENT  = Domoticz is a Home Automation System.
+LICENSE  = GPL-3.0
 
-CONFIGURE_TARGET = myConfigure
-INSTALL_TARGET = myInstall
+CONFIGURE_TARGET = domoticz_configure
+INSTALL_TARGET = domoticz_install
 
 BOOST_LIBRARIES = system thread date_time
```

Comment:
So it doesn't need the python boos-library? Adding `cross/python3` is enough?

Reply:
I don't know whether the python boost bindings are needed for the plugins, but so far python-boost was not created (and I didn't change BOOST_LIBRARIES for domitocz)